### ISSUE TITLE
Can't use ServiceAccountCredential

### ISSUE THREAD

**Issue by Aldo111** [appsup-dart/firebase_admin#1]
Hi, I'm trying to use this to verify a JWT token sent from my UI (which uses Firebase Auth) to my custom backend server and I can't seem to use ServiceAccountCredential as shown in the front example (doesn't seem available in namespace).

I thought I could maybe instead use Credentials.cert('path/to/my/private/key.json') but that seems unimplemented.

What would be the best approach here? (I'm trying to follow these instructions: https://firebase.google.com/docs/auth/admin/verify-id-tokens#nodejs)

**Comment by rbellens** [comment]
try 

```dart
FirebaseAdmin.instance.certFromPath('path/to/my/private/key.json');
```

**Comment by judeosbert** [comment]
try adding 
```
import 'package:firebase_admin/src/auth/credential.dart';
```
as import. It should give your `ServiceAccountCredential`

**Comment by akfaisel** [comment]
Where should the service-account.json be located? I'm using Windows 10. 

Please help me what is the actual path to the json file?

**Comment by rbellens** [comment]
I've extended the documentation of `Credentials.applicationDefault()` in version `0.1.3`. Additionally, `Credentials.applicationDefault()` now also looks for a `service-account.json` in the main directory of your package. Lastly, there was also a bug (see #3) in `FirebaseAdmin.instance.certFromPath` which is now fixed.

So, you now have the following options:

* load credential by using `FirebaseAdmin.instance.certFromPath`
* use `Credentials.applicationDefault()`, which will look for credentials at different locations:
    * the service-account.json file in the main directory of your package
    * the path specified in the env variable GOOGLE_APPLICATION_CREDENTIALS
    * credentials previously obtained by `Credentials.login` and stored at `Credentials.firebaseAdminCredentialPath`
    * credentials from the firebase-tools (when you have this installed)
